container-search/src/main/java/com/yahoo/search/query/ranking/WeakAnd.java

```diff
@@ -0,0 +1,83 @@
+// Copyright Vespa.ai. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+package com.yahoo.search.query.ranking;
+
+import com.yahoo.search.query.profile.types.FieldDescription;
+import com.yahoo.search.query.profile.types.QueryProfileType;
+
+import java.util.Objects;
+
+/**
+ * Query properties exposing WeakAnd-specific settings.
+ *
+ * @author vekterli
+ */
+public class WeakAnd implements Cloneable {
+
+    private static final QueryProfileType argumentType;
+
+    public static final String STOP_WORD_LIMIT = "stopwordLimit";
+    public static final String ADJUST_TARGET = "adjustTarget";
+
+    static {
+        argumentType = new QueryProfileType(Matching.WEAKAND);
+        argumentType.setStrict(true);
+        argumentType.setBuiltin(true);
+        argumentType.addField(new FieldDescription(STOP_WORD_LIMIT, "double"));
+        argumentType.addField(new FieldDescription(ADJUST_TARGET, "double"));
+    }
+
+    public static QueryProfileType getArgumentType() { return argumentType; }
+
+    private Double stopWordLimit = null;
```

Comment:
I think this should be _stopwordLimit_, _getStopwordLimit()_, and _setStopwordLimit()_ to match the name in the rank profile.

Reply:
Updated. `getStopwordLimit` et al just somehow _feel_ a bit more awkward than `getStopWordLimit` to me 🙈